go-controller/pkg/ovn/obj_retry.go

```diff
@@ -750,7 +754,7 @@ func (oc *Controller) updateResource(objectsToRetry *RetryObjs, oldObj, newObj i
 		gwSync := (failed || gatewayChanged(oldNode, newNode) ||
 			nodeSubnetChanged(oldNode, newNode) || hostAddressesChanged(oldNode, newNode))
 
-		return oc.addUpdateNodeEvent(newNode, &nodeSyncs{nodeSync, clusterRtrSync, mgmtSync, gwSync})
+		return oc.addUpdateNodeEvent(newNode, &nodeSyncs{nodeSync, clusterRtrSync, mgmtSync, gwSync, false})
```

Comment:
@JacobTanenbaum do we want to do what the rest of the nodeSyncs do and Load the HO failure here instead of assume false?

Reply:
done